_posts/2019-10-3-Amin.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Bob
```

Comment:
Please change the `title` to something unique, as the current one clashes with [this](https://github.com/sunwaytechclub/hacktoberfest-2019/blob/master/_posts/2019-9-18-Bob.md).